src/main/resources/io/jenkins/plugins/gitlabbranchsource/servers/Messages.properties

```diff
@@ -0,0 +1,11 @@
+GitLabServers.displayName=GitLab
+GitLabServer.hookManagementAs=Hook management will be performed as <code>{0} </code>
+GitLabServer.validateInterrupted=Interrupted while trying to validate credentials
+GitLabServer.versionInterrupted=Interrupted while trying to determine server version
+GitLabServer.invalidUrl=Invalid URL: <code>{0}</code>
+GitLabServer.serverVersion=GitLab Version: <code>{0}</code>
+GitLabServer.failedValidation=Failed to validate the account <code>{0}</code>
+GitLabServer.credentialsNotResolved=Cannot resolve suitable credentials with id: <code>{0}</code>
+GitLabServer.someException=Something failed cannot say what exactly, recheck server url and token
```

Comment:
I am sure this is not your wording but it sounds incorrect `GitLabServer.someException=Something failed cannot say what exactly, recheck server url and token` should maybe say something like `GitLabServer.someException=Something failed and we cannot say what exactly, recheck server url and token`

Reply:
That was an impromptu message, I have to figure out what's going wrong. Will update with your version and later fix it.